### ISSUE TITLE
Require virtualmin-nginx-ssl/virtual_feature.pl failed

### ISSUE THREAD

**Issue by chris001** [webmin/webmin#576]
```
Features and Plugins 	
This page allows you to select the core features and plugins that are available for use by virtual servers. The checkbox in the first column determines if it is available at all, while the box in the Default column controls whether or not it is enabled by default.

Require virtualmin-nginx-ssl/virtual_feature.pl failed :

Undefined subroutine &virtualmin_nginx_ssl::load_language called at /usr/share/webmin/virtualmin-nginx-ssl/virtual_feature.pl line 9.
```
Debian 8.
The deb packages `webmin-virtualmin-nginx` and `webmin-virtualmin-nginx-ssl` are installed.

**Comment by swelljoe** [comment]
Which versions of those packages do you have?

**Comment by chris001** [comment]
```
~# apt show webmin-virtualmin-nginx-ssl -a
Package: webmin-virtualmin-nginx-ssl
Essential: no
Priority: optional
Section: admin
Installed-Size: 331 MB
Maintainer: Virtualmin, Inc. <security@virtualmin.com>
Version: 1.5
Provides: webmin-virtualmin-nginx-ssl
Depends: base, perl, webmin, webmin-virtualmin-nginx, webmin-virtual-server
Pre-Depends: bash, perl
Download-Size: 63.2 kB
SHA512: 1240f7fad0cbcb46e0377f643eadc210a86e7ce050a7ade7b2028b0909c8435841fd041f39539e7358963855603d771db8c8dd71c775ed5cbf2675d58318fdda
APT-Manual-Installed: yes
APT-Sources: http://software.virtualmin.com/gpl/debian/ virtualmin-universal/main amd64 Packages
Description: Webmin module for 'Nginx Webserver SSL'

Package: webmin-virtualmin-nginx-ssl
Essential: no
Priority: optional
Section: admin
Installed-Size: 67.1 MB
Maintainer: Virtualmin, Inc. <security@virtualmin.com>
Version: 1.4
Provides: webmin-virtualmin-nginx-ssl
Depends: base, perl, webmin, webmin-virtualmin-nginx, webmin-virtual-server
Pre-Depends: bash, perl
Download-Size: 6860 B
SHA512: a214f9a262d144b49523dea9b897efcd850204db434106e0e7015ae0bfec3d2ae2068fdf6e2b5d1a2b3fd29a8f8ce7d3bccb756117a19fc32083c4f268ca10a0
APT-Sources: http://software.virtualmin.com/gpl/debian/ virtualmin-universal/main amd64 Packages
Description: Webmin module for 'Nginx SSL Webserver'

~# apt show webmin-virtualmin-nginx -a
Package: webmin-virtualmin-nginx
Essential: no
Priority: optional
Section: admin
Installed-Size: 2114 MB
Maintainer: Virtualmin, Inc. <security@virtualmin.com>
Version: 2.2
Provides: webmin-virtualmin-nginx
Depends: base, perl, webmin, webmin-virtual-server (>= 5.06)
Pre-Depends: bash, perl
Download-Size: 725 kB
SHA512: 90767951c048175f6a6e5d8d1bfc5633e3bada3525bafeb508015cd5d8498b62b4005ca7bde180611c8a0ed749258a4ca675fabd707645434cecb8c6b93b3338
APT-Manual-Installed: yes
APT-Sources: http://software.virtualmin.com/gpl/debian/ virtualmin-universal/main amd64 Packages
Description: Webmin module for 'Nginx Webserver'

Package: webmin-virtualmin-nginx
Essential: no
Priority: optional
Section: admin
Installed-Size: 2001 MB
Maintainer: Virtualmin, Inc. <security@virtualmin.com>
Version: 2.1
Provides: webmin-virtualmin-nginx
Depends: base, perl, webmin, webmin-virtual-server (>= 5.06)
Pre-Depends: bash, perl
Download-Size: 677 kB
SHA512: 058f811845112a0ec7736a13d0d046fe2e444a4b9e78f536985487cdcdb569fa78f64d990fc6a8a3a8b5ee1bf6a5536c8a841fc7c8096d0f69c26385b2bbbbc4
APT-Sources: http://software.virtualmin.com/gpl/debian/ virtualmin-universal/main amd64 Packages
Description: Webmin module for 'Nginx Webserver'
```

**Comment by jcameron** [comment]
Did you recently update anything, like the `webmin` or `virtual-server` packages?

**Comment by chris001** [comment]
IIRC apt recently updated `webmin`.
```
# apt list webmin -a
Listing... Done
webmin/stable,now 1.840 all [installed]
webmin/stable 1.831 all
webmin/stable 1.830 all

# apt list webmin-virtualmin-nginx-ssl -a
Listing... Done
webmin-virtualmin-nginx-ssl/stable,now 1.5 all [installed]
webmin-virtualmin-nginx-ssl/stable 1.4 all

# apt list webmin-virtualmin-nginx -a
Listing... Done
webmin-virtualmin-nginx/stable,now 2.2 all [installed]
webmin-virtualmin-nginx/stable 2.1 all

# apt list webmin-virtual* -a
Listing... Done
webmin-virtual-server/stable,now 5.07.gpl-3 all [installed]
webmin-virtual-server/stable 5.07.gpl all
webmin-virtual-server/stable 5.06.gpl-2 all
```

**Comment by jcameron** [comment]
Odd, on a fully updated CentOS 6 system with the same versions of all modules, I was unable to re-produce this.

**Comment by chris001** [comment]
It appears the cause of the problem was, the `webmin-virtualmin-nginx.deb` and `webmin-virtualmin-nginx-ssl.deb` files, downloaded directly from `virtualmin.com`, let you install them directly with `dpkg -i`.  And then also, `apt` installed two other ones, without realizing it was overwriting the existing ones from `dpkg`.  Then, the ones installed by `dpkg` were removed, yet the system thought the ones installed by `apt` were still installed, yet the one copy of the packages was uninstalled with already.  I fixed it by, 
```
apt remove webmin-virtualmin-nginx webmin-virtualmin-nginx-ssl
apt install webmin-virtualmin-nginx, webmin-virtualmin-nginx-ssl
```
